lib/github-service.js

```diff
@@ -29,6 +37,21 @@ class GithubService {
     });
   }
 
+  async createPullRequestComment(report){
+    const res = await getAssociatedPullRequestNumberFromSha(this.getOctokitInstance(), this.options.repo, this.options.refCommit);
```

Comment:
Here we should chain both method using then since they are dependent to each other. No need to use await

Reply:
Done